packages/openapi-ts/src/plugins/@pinia/colada/queryOptions.ts

```diff
@@ -152,9 +161,12 @@ export const createQueryOptions = ({
         async: true,
         multiLine: true,
         parameters: [
-          {
-            name: fnOptions,
-          },
+          isNuxtClient
+            ? {
+                name: fnOptions,
+                type: `Partial<${strippedTypeData}>`,
```

Comment:
Is this desirable? Won't this make everything optional, including potentially required fields like `body` or `path`?